### ISSUE TITLE
[Feature Request] Expose the build command as a `package.json` script.

### ISSUE THREAD

**Issue by ApacheTech** [theron-wang/VS2022-Editor-Support-for-Tailwind-CSS#5]
Would it be possible to expose the build command, through a  script in `package.json`? The name of the script can be set in the Build options, and it would be a `string.Format` template, where `{0}` is the input file, and `{1}` is the output file. That would give instant support for all kinds of things, like PostCSS, Concurrently, etc., and would also allow setting of environment variables. Anything you can do with the scripting system.

**Comment by theron-wang** [comment]
Yep, definitely. This would be a good feature to add. I will include this in the next update. Thanks!

**Comment by theron-wang** [comment]
I've just released an update which includes this feature, but I chose not to include the `string.Format` suggestion, since it is (to my knowledge) not possible to modify the `package.json` script value if I call `npm run`.

The current implementation should be able to reach the same result, just with a little more work - you just have to define the input and output files manually in `package.json`.

Thank you again for the suggestion!
